feats/feats.yml

```diff
@@ -839,19 +842,19 @@
   description: |
     Following a trail is an everyday part of your life, and as such it is no more taxing than breathing or blinking. Whether you are an experienced woodsman, a bounty hunter for the Galactic Senate, or a humanoid cat creature with heightened senses, your expertise at tracking allows you to get it done faster than the average hunter.
   effect: |
-    You can move at your full speed while tracking your quarry rather than the reduced speed typically incurred.
+    You move unhindered and at full speed when tracking your quarry.
 - !
   name: Ferocious Minions (I - III)
   prerequisites:
     tier1:
       Attribute:
-      - Influence: 3
+      - Invoking Attribute: 4
     tier2:
```

Comment:
Thanks for referencing the comment. I think that `Invoking Attribute` should be expanded to Creation, Entropy, Logic (the prerequisite for the `Animation` boon) and Influence (`charmed + dominated`). I don't see the logic behind Energy and Alteration, though (maybe I am missing something).